tests/asserts/final/publication/micka.py

```diff
@@ -0,0 +1,108 @@
+                                            tested=md_record,
+                                            )
+
+
+def correct_values_in_layer_metadata(workspace, publ_type, name, ):
+    assert publ_type == process_client.LAYER_TYPE
+    with app.app_context():
+        resp_json = process_client.get_workspace_layer_metadata_comparison(workspace, name,)
+        assert METADATA_PROPERTIES.issubset(set(resp_json['metadata_properties'].keys()))
+        for key, value in resp_json['metadata_properties'].items():
+            assert value['equal_or_null'] is True, f'key={key}, value={value}'
+
+        publ_info = layman_util.get_publication_info(workspace, publ_type, name)
+
+        native_bbox = publ_info.get('native_bounding_box')
+        crs = publ_info.get('native_crs')
+        if bbox_util.is_empty(native_bbox):
+            native_bbox = crs_def.CRSDefinitions[crs].default_bbox
+        extent = bbox_util.transform(native_bbox, crs_from=crs, crs_to=crs_def.EPSG_4326)
+        url = layman_util.url_for('rest_workspace_layer.get', layername=name, workspace=workspace)
+
+        file_type = publ_info['_file_type']
+        if file_type == settings.FILE_TYPE_VECTOR:
+            table_name = publ_info['db_table']['name']
+            try:
+                languages = db.get_text_languages(workspace, table_name)
+            except LaymanError:
+                languages = []
+            # try:
+            #     scale_denominator = db.guess_scale_denominator(workspace, table_name)
+            # except LaymanError:
+            #     scale_denominator = None
+            # spatial_resolution = {
+            #     'scale_denominator': scale_denominator,
+            # }
+        elif file_type == settings.FILE_TYPE_RASTER:
+            languages = []
+            # bbox_sphere_size = prime_db_table.get_bbox_sphere_size(workspace, name)
+            # distance_value = gdal.get_normalized_ground_sample_distance_in_m(workspace, name,
+            #                                                                  bbox_size=bbox_sphere_size)
+            # spatial_resolution = {
+            #     'ground_sample_distance': {
+            #         'value': distance_value,
+            #         'uom': 'm',
+            #     }
+            # }
+
+    exp_metadata = {'abstract': publ_info['description'],
+                    'extent': list(extent),
+                    'graphic_url': publ_info['thumbnail']['url'],
+                    'identifier': {
+                        'identifier': url,
+                        'label': name,
+                    },
+                    'layer_endpoint': url,
+                    'language': languages,
+                    'organisation_name': None,
+                    'publication_date': date.today().strftime('%Y-%m-%d'),
+                    # Revision date is empty after patch (https://trello.com/c/hRkFNuXP)
+                    # 'revision_date': date.today().strftime('%Y-%m-%d'),
```

Comment:
Test also revision_date value, that differs on patch/post. Get method from rest_method parameter.

Reply:
Comparing rest_method with TestSingleRestPublication.post_publication/patch_publication methods.